test/test_importing.py

```diff
@@ -40,3 +40,11 @@ def test_importing_attributes(self):
         assert mod.__spec__.loader is mod.__loader__
         assert mod.__spec__.name == 'mytest'
         assert mod.__file__
+
+        if hpy_abi == 'cpython':
+            from distutils.sysconfig import get_config_var
```

Comment:
```suggestion
            from sysconfig import get_config_var
```

Reply:
See also https://bugs.python.org/issue39825. `distutils` is officially deprecated, anything hpy needs should be imported from `sysconfig` or `setuptools`